src/ansys/simai/core/utils/typing.py

```diff
@@ -20,21 +20,21 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-from typing import TYPE_CHECKING, cast
+import typing as t
```

Comment:
Convenient but all the `t.{type}`s makes the thing less readable IMO

Reply:
I agree on this. I'd prefer either to import it without an alias or import just the required functions (as we generally do in this repo).